theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/shared.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * This 4-tuple defines the start control point x1,y1 and the end control point x2,y2
+ * of a cubic bezier curve. It is assumed that the start of the curve is fixed at 0,0
+ * and the end is fixed at 1,1. x1 must always be >= 0 and x2 must always be <= 1.
+ *
+ * to get a feel for it: https://cubic-bezier.com/
+ **/
+export type CubicBezierHandles = [
+  x1: number,
+  y1: number,
+  x2: number,
+  y2: number,
+]
+
+export function cssCubicBezierArgsFromHandles(points: CubicBezierHandles) {
+  return points.map((p) => p.toFixed(2)).join(', ')
+}
```

Comment:
Change: It was pointed out this should be `p.toFixed(3)`, IIUC.

Reply:
Done